event/models.py

```diff
@@ -23,6 +23,12 @@ class EventSponser(models.Model):
     def __str__(self):
         return self.name
 
+class EventTag(models.Model):
+    name = models.CharField(max_length=255)
```

Comment:
should we have name and label as separate? @abdulmoizeng

Reply:
i would suggest **label & description** will look more meaningful here